zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -111,7 +111,9 @@ zipkin:
     # - .*example2.com
     # Default is "match all websites"
     instrumented: .*
-
+    # Url of the logs associated with a given request.
+    # It will be formatted with the trace id if your url contains ${traceId}} (e.g. http://kibana/trace${traceId})
+    logsUrl: ${LOGS_URL:}
```

Comment:
Small nit-pick: other variables are kebab-case in this file, so it might match the style better if this were too. Thanks to Spring Boot's relaxed property binding, it shouldn't effectively matter either way, though.